docker-compose.test.yml

```diff
@@ -7,6 +7,7 @@ services:
     command: ["go", "test", "-v", "github.com/appcelerator/amp/data/etcd"]
```

Comment:
Yeah we should be running `make test` and keeping make test up to date.

I don't see a good reason to make a separate `make test-docker` the local docker flow is to use `docker-compose -f docker-compose.test.yml` to bring up the backing services and run the sut.

Which means we'll need to keep `docker-compose.test.yml` up to date, and it should contain all services needed to test